static/usage/modal/can-dismiss/boolean/vue.md

```diff
@@ -41,6 +41,7 @@
     IonLabel,
     IonPage,
   } from '@ionic/vue';
+  import { CheckboxCustomEvent } from '@ionic/core/components';
```

Comment:
This is exported from `@ionic/vue` too: https://github.com/ionic-team/ionic-framework/blob/484de5074de212dffb4bf4f73bade7acec103fe8/packages/vue/src/index.ts#L79